src/js/common/schemaform/review/ObjectField.jsx

```diff
@@ -51,20 +51,25 @@ class ObjectField extends React.Component {
     const orderedProperties = orderProperties(properties, uiSchema['ui:order']);
     const isRoot = idSchema.$id === 'root';
 
-    const renderedProperties = orderedProperties.map((propName, index) => {
-      return (
-        <SchemaField key={index}
-            name={propName}
-            schema={schema.properties[propName]}
-            uiSchema={uiSchema[propName]}
-            errorSchema={errorSchema[propName]}
-            idSchema={idSchema[propName]}
-            onChange={f => f}
-            onBlur={f => f}
-            formData={formData[propName]}
-            registry={this.props.registry}/>
-      );
-    });
+    const renderedProperties = orderedProperties
```

Comment:
This is very hard for me to read and decipher what exactly this is checking. Any way we can make this is a bit more obvious? Some comments in the code might help here.